### PR TITLE
Translated some public-facing strings in the portal popup

### PR DESCRIPTION
**Pull request by benjaminrancourt**
refs https://github.com/TryGhost/Ghost/issues/16628

I made only these changes as I want to quickly see if my local environment was correctly set up. 

Only French texts have been translated in this commit. Is it ok? 🤔

Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [x] There's a clear use-case for this code change, explained below
- [x] Commit message has a short title & references relevant issues
- [x] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

---

### <samp>🤖 Generated by Copilot at 4091ea2</samp>

This pull request updates and adds translations for the portal feature in 18 different languages. It modifies the `./ghost/i18n/locales/[language]/portal.json` files and the `./ghost/i18n/locales/context.json` file to improve the localization and user experience of the portal interface for various speakers.

### CODE REVIEW

**Comment by benjaminrancourt** [comment]
image

I will mark this pull request as draft because I pushed a more complex case which is currently displaying incorrectly. If anyone knows how to fix it, it will help me complete this pull request! 🥰

File: ghost/portal/src/components/pages/EmailReceivingFAQ.js [hunk 1]
```diff
@@ -24,31 +24,31 @@ export default function EmailReceivingPage() {
             </header>
 
             <div class="gh-longform">
-                <h3>Help! I'm not receiving emails</h3>
+                <h3>{t("Help! I'm not receiving emails")}</h3>
```
Comment (daniellockyer) [inline ghost/portal/src/components/pages/EmailReceivingFAQ.js:27]:
```suggestion
                <h3>{t('Help! I\'m not receiving emails')}</h3>
```

**Comment by benjaminrancourt** [comment]
> For your tricky example, you should be able to do something like:
> 
> ```js
> <p>{t('If you have a corporate or government email account, reach out to your IT department and ask them to allow emails to be received from {email}', {email: `<strong>${defaultNewsletterSenderEmail}</strong>`, interpolation: {escapeValue: false}})}</p>
> ```

I tried many attempts, but none worked. I don't know React very well, so it would be great if someone how has translated a React application before could help us! 😅

Some attempts that I've tried: 
```js
                <p>A - <strong>{defaultNewsletterSenderEmail}</strong></p>
                <p>1 - {t('{email}', {email: `<strong>${defaultNewsletterSenderEmail}</strong>`, interpolation: {escapeValue: false}})}</p>
                <p>2 - {t('{email}', {email: `<strong>${defaultNewsletterSenderEmail}</strong>`, interpolation: {escapeValue: true}})}</p>
                <p>3 - {t('{{email}}', {email: `<strong>${defaultNewsletterSenderEmail}</strong>`, interpolation: {escapeValue: false}})}</p>
                <p>4 - {t('{{email}}', {email: `<strong>${defaultNewsletterSenderEmail}</strong>`, interpolation: {escapeValue: true}})}</p>

                <p>5 - {t('{- email}', {email: `<strong>${defaultNewsletterSenderEmail}</strong>`, interpolation: {escapeValue: false}})}</p>
                <p>6 - {t('{- email}', {email: `<strong>${defaultNewsletterSenderEmail}</strong>`, interpolation: {escapeValue: true}})}</p>
                <p>7 - {t('{{- email}}', {email: `<strong>${defaultNewsletterSenderEmail}</strong>`, interpolation: {escapeValue: false}})}</p>
                <p>8 - {t('{{- email}}', {email: `<strong>${defaultNewsletterSenderEmail}</strong>`, interpolation: {escapeValue: true}})}</p>
```

And the screenshot of the results:
image

Anyone reading this have an idea? 🤔

**Comment by benjaminrancourt** [comment]
I also tried using the [`<Trans>` component ](https://react.i18next.com/latest/trans-component) from [react-i18next](https://www.npmjs.com/package/react-i18next), but the text doesn't seem to be fetched by i18next/i18next-parser... 🙈

```js
import { Trans } from 'react-i18next'

// ***

 <Trans t={t} i18nKey="test">
   <p>If you have a corporate or government email account, reach out to your IT department and ask them to allow emails to be received from <strong>{defaultNewsletterSenderEmail}</strong></p>
</Trans>
```

**ghost/i18n/lib/i18n.js**
```js
const { initReactI18next } = require("react-i18next");

// ..

    const i18nextInstance = i18next.createInstance();
    i18nextInstance.use(initReactI18next)
    .init({
        lng,

// ...

**Pull request closed without merging**